pythainlp/transliterate/spoonerism.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+from pythainlp.transliterate import pronunciate
+from pythainlp import thai_consonants
+
+_list_consonants = list(thai_consonants.replace("ห", ""))
+
+
+def puan(word: str, show_pronunciation: bool = True) -> str:
+    """
+    Thai Spoonerism
+
+    This function covent Thai word to Thai spoonerism word.
+    This function only supports words with 2 to 3 syllables.
+
+    :param str word: Thai word to be spoonerism
```

Comment:
Thai word to be (spoonerized). This should be V3. `spoonerism` sounds like a noun to me.